README.md

```diff
@@ -139,11 +147,11 @@ Our Slack Community: [Slack Invite](http://bit.ly/NITRDevs) <br>
 
 Please refer to the project's style and contribution guidelines for submitting patches and additions. In general, we follow the "fork-and-pull" Git workflow.
 
-1.  **Fork** the repo on GitHub
-2.  **Clone** the project to your local system
-3.  **Commit** changes to your own separate branch
-4.  **Push** your work back up to your fork
-5.  Submit a **Pull request** so that we can review your changes
+1. **Fork** the repo on GitHub
+2. **Clone** the project to your local system
+3. **Commit** changes to your own separate branch
+4. **Push** your work back up to your fork
+5. Submit a **Pull request** so that we can review your changes
```

Comment:
Why is this happening at multiple places?

Reply:
fixed https://github.com/dscnitrourkela/project-udon/pull/11/commits/90eb246dc7a65683019745a4905d17222c6a949b